frontend/wallet/src/render/views/settings/AccountSettings.re

```diff
@@ -332,7 +337,7 @@ module BlockRewards = {
                                     onMouseLeave={_ =>
                                       setStakingHovered(_ => false)
                                     }
-                                    onClick={_ => mutate() |> ignore}
+                                    onClick={_ => Task.liftPromise(mutate) |> Task.perform(~f=_ => {Bindings.setTimeout(1000) |> ignore response.refetch(Some(accountInfoVariables))} |> ignore) }
```

Comment:
Does it need 1000? Can you test 100 -- and see if that works (or even 200 or 300), I think 1000 makes the UI update a little jarring (but worst case is fine for now)

Reply:
100 is also ok!